TodoList/Src/TodoList.Contracts/DI/ITodoListDependencyResolver.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Web.Http.Dependencies;
+
+namespace TodoList.Contracts.DI
+{
+    public interface ITodoListDependencyResolver : IDependencyResolver
```

Comment:
Why?

Reply:
Because you don't want to be dependant on `System.Web`.

You can use "this" without frontend (in another application) and then you definitely don't want a reference to `System.Web`.

Also, an empty interface is a code smell. 
![tumblr_nok6hoaqze1r1n25go4_500](https://user-images.githubusercontent.com/40827092/49443160-fd9ac300-f7cb-11e8-9f7d-5c48738a5457.gif)

Reply:
Thanks for the description. Now I make sense. 

Should I delete extend to `IDependencyResolever` and add methods to my interface?

```c#
object GetService(Type serviceType);

IEnumerable<object> GetServices(Type serviceType);
```
+ `IDisposable`

Or I can throw out the whole interface?

Reply:
e928364